20/android/Dockerfile

```diff
@@ -0,0 +1,49 @@
+# Default image that can build OpenRCT2 for Android.
+FROM ubuntu:24.04
+RUN apt-get update && \
+    apt-get -y upgrade && \
+    DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -y \
+  # System
+    ca-certificates curl unzip \
+  # Build tools
+    git cmake pkg-config ninja-build ccache g++ \
+  # Sprite builder libraries for the host
+    libpng-dev \
+    openjdk-21-jdk
+
+ENV ANDROID_HOME=/android-dev
+ENV ANDROID_NDK_HOME="$ANDROID_HOME/ndk"
+ENV JAVA_HOME=/usr/lib/jvm/java-21-openjdk-amd64
+
+# Grab the Android SDK
+WORKDIR /tmp/setup
+RUN curl -Lo sdk.zip https://dl.google.com/android/repository/commandlinetools-linux-13114758_latest.zip \
+ && unzip -qo sdk.zip \
+ && mkdir -p $ANDROID_HOME/cmdline-tools \
+ && mv cmdline-tools $ANDROID_HOME/cmdline-tools/latest \
+ && rm sdk.zip
+
+# Need nlohmann/json, needs to live outside system libraries, so it doesn't
+# confuse toolchain to include other host headers
+RUN curl -Lo json.zip https://github.com/nlohmann/json/releases/download/v3.11.3/include.zip \
```

Comment:
Somewhere you mentioned why we can't use 3.12 yet, please add this note here as well

Reply:
I copied my comment from https://github.com/OpenRCT2/Dependencies/pull/44 to just after the existing comment.